Script/AtomicNET/AtomicNET/Core/Profiler.cs

```diff
@@ -12,12 +13,16 @@ public partial class Profiler : AObject
         {
 #if ATOMIC_PROFILING
             var profiler = AtomicNET.GetSubsystem<Profiler>();
-            profiler?.BeginBlock(name, file, line, color, (byte)status);
+            if (profiler != null)
```

Comment:
BTW, will this be the first C# code profiled with easy_profiler @cas4ey, awesome work again @rokups 🥇